.rules

```diff
@@ -1,30 +1,137 @@
+# System Prompt
```

Comment:
I would like to see this entire section removed from the file. I think this belongs into a user-level file, and the checked-in rules file should only contain information relevant to the project iself.

As-is, this file is heavily in conflict with my own setup.